### DISCUSSION TITLE
fields available in global env?

### DISCUSSION THREAD

**Discussion by nferch** [goreleaser/goreleaser#3902] [Q&A] [answered]
Fields do not appear to be available to templates used in the global env section, as I was led to believe at https://goreleaser.com/customization/templates/

e.g.
```
env:
  - MYBRANCH={{ .Branch }}-test
```

results in MYBRANCH being set to "-test".

I'm new to GoReleaser so it's possible I'm missing something here.

**Comment by caarlos0** [comment, marked as answer]
the `env` part is one of the first ones to run, before goreleaser knows against which branch it is running, that's why it didn't work.
